Tasks/PublishSymbols/PublishSymbols.ps1

```diff
@@ -64,15 +71,14 @@ Write-Verbose "maxWaitTime = $maxWaitTime"
 $maxSemaphoreAge = [timespan]::FromDays(1)
 Write-Verbose "maxSemaphoreAge = $maxSemaphoreAge"
 
-# The symbols search folder defaults to source folder. Override if symbolsFolder is passed
-$symbolsSearchFolder = $env:Build_SourcesDirectory
```

Comment:
I added unit tests for this logic, so I took the opportunity to remove this extra variable that hinder readability.